src/main/java/mate/academy/model/Book.java

```diff
@@ -0,0 +1,52 @@
+package mate.academy.model;
+
+import java.math.BigDecimal;
+
+public class Book {
+    private Long id;
+    private String title;
+    private BigDecimal price;
+
+    public Book(String title, BigDecimal price) {
+        this.title = title;
+        this.price = price;
+    }
+
+    public Book(Long id, String title, BigDecimal price) {
+        this(title, price);
+        this.id = id;
+    }
```

Comment:
I would recommend to delete this constructor and initialize object properties with setters. 
It will add more consistency to your code, but it is optional.

Reply:
At the lecture, it was said that it is more correct to create a new constructor than to use the previous one and set id additionally.